src/bci_build/package/base/README.md.j2

```diff
@@ -5,24 +5,67 @@
+different environments. These container images address modern, containerized
+application development and CI/CD application containerization. They can be
 used immediately by developers and integrators without the “lock-in” imposed by
 other offerings.
 
 SLE BCI inherits industry-leading security and compliance from SUSE Linux
-within your container build process. The container images are designed to be a
-secure base for any application workload. SUSE ensures that compliance
-standards are applied consistently and continuously improves security-related
-capabilities.
-
-SLE BCI is lightweight and easy to adopt, with the ability to run with any
-Linux OS. Avoid lock-in imposed by other vendors and get exactly what you need,
-fast. SLE BCI delivers a flexible developer experience that accounts for,
-integrates with, and supports language-native tools and workflows.
+Enterprise within your container build process. The container images are
+designed to provide a secure base for any application workload. SUSE ensures
+that compliance standards are applied consistently and continuously improves
+security-related capabilities.
+
+SLE BCI is lightweight and easy to adopt, with the ability to run using any OCI
+compatible container runtime. Avoid lock-in imposed by other vendors and get
+exactly what you need. SLE BCI delivers a flexible developer experience
+that accounts for, integrates with, and supports language-native tools and
+workflows.
+
+
+## Usage
+
+This container image ships the `zypper` package manager, the free `SLE_BCI`
+repository and the `container-suseconnect` utility, which can give you access to
+the full SLES repositories with a valid SLES subscription. The image is intended
+to be extended by installing additional packages fitting your specific
+use-case.
+
+To build a custom derived image using a `Containerfile` which includes a C
+compiler, create the following `Containerfile`:
+```Dockerfile
+FROM {{ image.pretty_reference }}
+RUN set -euo pipefail; \
+    zypper -n ref; \
+    zypper -n in gcc; \
+    zypper -n clean; \
+    rm -rf /var/log/{lastlog,tallylog,zypper.log,zypp/history,YaST2}
+```
+And build the container using `buildah`:
+```bash
+buildah bud -t bci-gcc .
+```
+
+### The SLE_BCI repository
+
+The free `SLE_BCI` repository is pre-installed in this container image. The
+repository contains a subset of all packages from SUSE Linux Enterprise Server,
+free of charge, but without any support. It also contains only the latest
+version of packages and not any previous versions or builds.
+
+
+### Getting access to the SLES repositories
+
+This image includes the `container-suseconnect` utility, which can automatically
+add the full SUSE Linux Enterprise Server repositories into the running
+container if provided with a valid SLES subscription. Find more information
+about container-suseconnect in the ["How to use
+container-suseconnect"](https://opensource.suse.com/bci-docs/guides/container-suseconnect/)
+guide.
```

Comment:
I think we're overly focusing on that there is a utility preinstalled. we should more focus on how to  use the feature than how it was implemented. so the important part is "how to use registration" and there might be a section in the container guide about it.. (not sure which documentation is the better reference for it)